### ISSUE TITLE
Remove an environment variable to multiple applications, devices, or services

### ISSUE THREAD

**Issue by nwneisen** [balena-io/balena-cli#1957]
This is a previously discussed issue that is desired to be added to the cli.

# Expected Behavior
Remove an environment variable to multiple applications, devices, or services with one command.

# Actual Behavior
Not currently implemented

# Steps to Reproduce
`balena env rm -d <uuid>,<uuid>...`
`balena env rm -s <name>,<name>...`
`balena env rm -a <name>,<name>...`

# Additional References
Originally mentioned in #323.

**Comment by nwneisen** [comment]
I'll do this after #1956 and follow similar pattern for the questions asked there.

**Comment by rdecarreau** [comment]
Is this going to happen? I have 74 devices out of a very large fleet and I need to remove an override from those specific devices.

**Comment by myarmolinsky** [comment]
> Is this going to happen? I have 74 devices out of a very large fleet and I need to remove an override from those specific devices.

Hi, while we still think this would be nice to have, it isn't currently on our roadmap to be worked on. So for now unfortunately you would need to do a loop over a list and remove env vars individually

**Comment by rdecarreau** [comment]
Thanks! Any way to do that somewhat quickly without getting rate throttled by the Balena API?

**Comment by myarmolinsky** [comment]
Let me check with our API team for their suggestions, I'll come back to you once I've heard back from them
